### PR TITLE
use app flow session endpoints and cleanup

### PR DESCRIPTION
**Pull request by xavluiz**
* this updates the flow mode enable/disable endpoints to point to the app server
* also minor cleanup of unused functions and arguments

### CODE REVIEW

File: src/managers/FlowManager.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export async function initiateFlow({automated = false, skipSlackCheck = false})
     // only update flow change here
     updateFlowChange(true);
     logIt('Entering Flow Mode');
-    await softwarePost('/v1/flow_sessions', {automated}, getItem('jwt'));
+    await appPost('/v1/flow_sessions', {automated});
```
Comment (bjacobson26) [inline src/managers/FlowManager.ts:82]: wont this go to: `/plugin/flow_session`?

Comment (xavluiz) [reply src/managers/FlowManager.ts:82]: ah yes, i had that in my testing, then reverted it for something else. thanks!

Comment (xavluiz) [reply src/managers/FlowManager.ts]: thanks again. updated

File: src/notifications/endOfDay.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const setEndOfDayNotification = async (user: any) => {
       msUntilEndOfTheDay = getMillisUntilEndOfTheDay(d, HOUR_IN_MILLIS * 17);
     }
 
-    const response = await softwareGet('/users/profile', jwt);
+    const response = await softwareGet('/users/profile');
```
Comment (bjacobson26) [inline src/notifications/endOfDay.ts:35]: is this still going to codemetrics? should this also grab profile info from the software app?

Comment (xavluiz) [reply src/notifications/endOfDay.ts:35]: I haven't tested this. I'll do that now. Yeah, it'll be good to swap out what we can.

Comment (xavluiz) [reply src/notifications/endOfDay.ts:35]: It's not returning json, just html. We'll have to handle the client request for this.

**Review by mmclead — APPROVED** [review]

**Pull request merged**